lms/djangoapps/labster_course_license/utils.py

```diff
@@ -105,8 +106,13 @@ def course_tree_info(store, simulations, licensed_simulations):
     Retuns information about the course's xblocks.
     """
     course_info = {}
+    invalid_simulations = []
+    validation_pattern = r'^https:\/\/[\w\d]*\.labster\.com\/[a-z]*\/[\w\d]*\/$'
```

Comment:
You could split by `'/simulation/'` string and get the server string and `simulation_id + '/'`